digitiser-aggregator/src/main.rs

```diff
@@ -162,16 +197,27 @@ async fn on_message(
                             .await;
                         }
                         Err(e) => warn!("Invalid Metadata: {e}"),
+                        //  NOTE: Should any metrics be recorded here?
                     }
                 }
                 Err(e) => {
                     warn!("Failed to parse message: {}", e);
+                    counter!(
+                        FAILURES,
+                        &[failures::get_label(FailureKind::UnableToDecodeMessage)]
+                    )
+                    .increment(1);
                 }
             }
         } else {
             warn!("Unexpected message type on topic \"{}\"", msg.topic());
             debug!("Message: {msg:?}");
             debug!("Payload size: {}", payload.len());
+            counter!(
+                MESSAGES_RECEIVED,
+                &[messages_received::get_label(MessageKind::Unknown)]
```

Comment:
This category should probably be called "unexpected" rather than "unknown".

This could be a known message type, but just one that we did not expect to see.